### PR TITLE
docs(35/km/super-admin) create super admin post installation

### PR DESCRIPTION
**Pull request by nragokong**
Instructions on how to create a super admin and enable kong manager gui auth after Kong installation where kong password is not set (no default kong-admin created). Customers are frequently starting evaluations w/out setting up a default kong super admin and are struggling creating and enabling a super admin post installation. This update provides more specific details on how to do so that we can send to evaluators when needed.

**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md

### Summary

Customers are frequently starting evaluations w/out setting up a default kong super admin and are struggling creating and enabling a super admin post installation. This update provides more specific details on how to do so that we can send to evaluators when needed.

### Full changelog

* Edited how to create superadmin page

### Issues resolved

Fix #XXX

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation 
- [x] [Adheres to Kong style guide](https://github.com/Kong/docs.konghq.com/blob/master/STYLEGUIDE.md)
- [x] Spellchecked my updates
- [x] Tagged "Team Docs" as reviewers
- [x] Ready to be merged

### CODE REVIEW

**Comment by levfishbluefish** [comment]
@nragokong @h3xar0n 

I suggest creating a new page in the "Getting Started" section titled "How to Create a Super Admin"

Thoughts?

**Comment by nragokong** [comment]
@levfishbluefish I was going to do that then I realized we had a page titled that under Kong Manager/Authentication and didn't want to replicate so I added it there. I think we are falling into this because customers are following the enterprise install instructions (see dock or AWS linux for enterprise) which does not discuss the super admin bootstrap command for migrations (or setting the password as env variable). We may want to update that to recommend they set the password variable. Once they get to the getting started docs typically they already went to the install docs and stood up kong. Thoughts?

**Comment by h3xar0n** [comment]
@nragokong it seems like the solution would be to funnel them from each installation guide to the getting started section, which does mention the process and the caveat that it must be done during migrations: https://docs.konghq.com/enterprise/0.35-x/getting-started/start-kong/#step-1

That was the intended flow, do you think it would be adequate to just direct them there?

**Comment by levfishbluefish** [comment]
@h3xar0n can we go a step further and whenever we mention running migrations add a warning which links to the caveat mentioned in getting-started?

**Comment by h3xar0n** [comment]
Incidentally, this PR pertains to users who have already missed the step; I think it's in the right section and it should go in. 

Making that step easier to spot for new users is a separate but related concern.

**Comment by h3xar0n** [comment]
@levfishbluefish IIRC they are only mentioned once, in getting started, precisely to avoid the problem.

**Comment by h3xar0n** [comment]
@levfishbluefish my correction: they should only be mentioned in one place to avoid the problem. We previously had them in Kong Manager docs as well, and needed to remove them to avoid conflict with the main Getting Started section. Do you agree we should funnel them to one page?

**Comment by levfishbluefish** [comment]
Got it. Let's merge this in as is

**Comment by h3xar0n** [comment]
Created a separate issue for it, @levfishbluefish : https://github.com/Kong/docs.konghq.com/issues/1286

Will review the contents of this PR since it would still be relevant.

File: app/enterprise/0.35-x/kong-manager/authentication/super-admin.md [hunk 1]
```diff
@@ -29,4 +29,16 @@ click the email address of the user in order to view them.
 
 5. Click "Generate Registration Link". 
 
-6. Copy the link for later use after completing the account setup.
\ No newline at end of file
+6. Copy the link for later use after completing the account setup.
+
+## How to Create Your First Super Admin Account Post Installation
+
+In the event that the default kong_admin Super Admin was not seeded during the initial database preparation step as defined in [Step 1 in How To Start Kong Enterprise Securely](../../getting-started/start-kong/#step-1), the following steps outline how to create and enable a new Super Admin post installation.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/kong-manager/authentication/super-admin.md]:
```suggestion
In the event that the default `kong_admin`, **Super Admin**, was not seeded during the initial database preparation step as defined in [Step 1 in How To Start Kong Enterprise Securely](../../getting-started/start-kong/#step-1), the following steps outline how to create and enable a new Super Admin post installation. 
```

File: app/enterprise/0.35-x/kong-manager/authentication/super-admin.md [hunk 2]
```diff
@@ -29,4 +29,16 @@ click the email address of the user in order to view them.
...
 
-6. Copy the link for later use after completing the account setup.
\ No newline at end of file
+6. Copy the link for later use after completing the account setup.
+
+## How to Create Your First Super Admin Account Post Installation
+
+In the event that the default kong_admin Super Admin was not seeded during the initial database preparation step as defined in [Step 1 in How To Start Kong Enterprise Securely](../../getting-started/start-kong/#step-1), the following steps outline how to create and enable a new Super Admin post installation. 
+
+1. Follow the instructions outlined above to create a new ***Super Admin** user account and generate a registration link.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/kong-manager/authentication/super-admin.md]:
```suggestion
1. Follow the instructions outlined above to create a new **Super Admin** user account and generate a registration link.
```

File: app/enterprise/0.35-x/kong-manager/authentication/super-admin.md [hunk 3]
```diff
@@ -29,4 +29,16 @@ click the email address of the user in order to view them.
...
+
+2. Before the link generated above can be used, RBAC and GUI Authentication must be enabled. Follow the instructions on [how to enable Basic Auth on Kong Manager](../basic).
+
+3. Paste the URL in your browser and you will be asked to create a password for the newly defined ***Super Admin*** user on the Kong Manager. ***Please Note:*** In addition to the Kong Manager UI, this password will also be used as the Kong-Admin-Token needed to utilize the Kong Admin API.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/kong-manager/authentication/super-admin.md]:
```suggestion
3. Paste the URL in your browser and you will be asked to create a password for the newly defined **Super Admin** user on the Kong Manager. 

**Note:** In addition to Kong Manager, this password will also be used as the `Kong-Admin-Token` needed to utilize the Admin API.
```

**Comment by h3xar0n** [comment]
@levfishbluefish , implementing style guide changes directly, will merge when done. Sound good?

File: app/enterprise/0.35-x/kong-manager/authentication/super-admin.md [hunk 4]
```diff
@@ -29,4 +29,18 @@ click the email address of the user in order to view them.
 
 5. Click "Generate Registration Link". 
 
-6. Copy the link for later use after completing the account setup.
\ No newline at end of file
+6. Copy the link for later use after completing the account setup.
+
+## How to Create Your First Super Admin Account Post Installation
+
+In the event that the default `kong_admin`, **Super Admin**, was not seeded during the initial database preparation step as defined in [Step 1 in How To Start Kong Enterprise Securely](../../getting-started/start-kong/#step-1), the following steps outline how to create and enable a new Super Admin post installation.
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/kong-manager/authentication/super-admin.md]: Relative paths do not reflect the structure of how the site is built, so it 404s

```suggestion
In the event that the default `kong_admin`, **Super Admin**, was not seeded during the initial database preparation step as defined in [Step 1 in How To Start Kong Enterprise Securely](/enterprise/{{page.kong_version}}/getting-started/start-kong/#step-1), the following steps outline how to create and enable a new Super Admin post installation. 
```

File: app/enterprise/0.35-x/kong-manager/authentication/super-admin.md [hunk 5]
```diff
@@ -29,4 +29,18 @@ click the email address of the user in order to view them.
...
\ No newline at end of file
+6. Copy the link for later use after completing the account setup.
+
+## How to Create Your First Super Admin Account Post Installation
+
+In the event that the default `kong_admin`, **Super Admin**, was not seeded during the initial database preparation step as defined in [Step 1 in How To Start Kong Enterprise Securely](../../getting-started/start-kong/#step-1), the following steps outline how to create and enable a new Super Admin post installation. 
+
+1. Follow the instructions outlined above to create a new **Super Admin** user account and generate a registration link.
+
+2. Before the link generated above can be used, RBAC and GUI Authentication must be enabled. Follow the instructions on [how to enable Basic Auth on Kong Manager](../basic).
```
Comment (h3xar0n) [inline app/enterprise/0.35-x/kong-manager/authentication/super-admin.md]: Not a 404, but will change in order to keep consistent even if we change the path in the future:

```suggestion
2. Before the link generated above can be used, RBAC and GUI Authentication must be enabled. Follow the instructions on [how to enable Basic Auth on Kong Manager](/enterprise/{{page.kong_version}}/kong-manager/authentication/basic).
```

**Pull request merged**